src/server/index.js

```diff
@@ -204,6 +205,32 @@ nextApp.prepare().then(() => {
     }
   });
 
+  server.post('/profile/save', async (req, res) => {
+    const id = get(req, 'body.id');
+    const accessToken = get(req, 'session.passport.user.accessToken');
+    if (!accessToken) {
+      res.setHeader('Cache-Control', 's-maxage=3600, max-age=0');
```

Comment:
Why would we need that?

Reply:
No we don't need it, something I forgot to clean up.